x-pack/auditbeat/module/system/host/host.go

```diff
@@ -69,68 +69,68 @@ func (action eventAction) String() string {
 
 // Host represents information about a host.
 type Host struct {
-	info types.HostInfo
+	Info types.HostInfo
 	// Uptime() in types.HostInfo recalculates the uptime every time it is called -
 	// so storing it permanently here.
-	uptime time.Duration
-	ips    []net.IP
-	macs   []net.HardwareAddr
+	Uptime time.Duration
+	Ips    []net.IP
```

Comment:
This should have been `IPs` and `MACs`. I'm surprised that Hound CI didn't complain.